### ISSUE TITLE
Ability to disable automatic update checking

### ISSUE THREAD

**Issue by leumasme** [Aliucord/Aliucord#353]
### What is it that you'd like to see?

Add the ability to disable update checking. Currently, there is only the ability to toggle auto update, but even if it is off, you will still get update notifications.

Let us simply disable all automatic update checking
https://github.com/Aliucord/Aliucord/blob/120e9416c687706a1b77e78833c6ba28e4dc82ac/Aliucord/src/main/java/com/aliucord/Main.java#L363

### Request Agreement

- [X] I did indeed check to make sure my feature request is original!

**Comment by mantikafasi** [comment]
maybe geting it checked on every launch is bad but I wouldnt really suggest to disable auto updates completely

**Comment by leumasme** [comment]
> I wouldnt really suggest to disable auto updates completely

I don't personally see the need for automatic updates for an application like aliucord for the way I use it - It's not going to get any more broken unless discord decides to break old app versions, and I'm personally satisfied with the current state of the app and plugins, thus the automatic update check is just a chore for me right now. The ability to manually update would remain of course, in case an update is ever needed for some reason.

**Comment by rushiiMachine** [comment]
> I wouldnt really suggest to disable auto updates completely

Up to the induvidual user

**Comment by mantikafasi** [comment]
you are right but I also dont want some idiots (not you) to disable it mindlessly and come to support channel crying why their plugin is buggy
